### ISSUE TITLE
flickr 30k features download

### ISSUE THREAD

**Issue by ziyanyang** [jnhwkim/ban-vqa#44]
Are the hdf5 files in the downloaded flickr30k_features.zip used to reproduce the results? I don't see tsv files in flickr30k_features.zip but I do need the features and bounding boxes for flickr 30k validation/testing sets. The files in flickr30k_features.zip are confusing, for example, in val.hdf5 file, there are (30722, 2048) features, but in adaptive_detection_features_converter.py, known_num_boxes for validation set is 29906, so what are these 30722 features?

**Comment by jnhwkim** [comment]
Please refer to #43 for the context. 

In short, I believe 30722 is from July 2018 (or earlier and the camera-ready version I believe), and 29906 is from @jaesuny 's reproduction in Nov 2018. The discrepancy is from the adaptive number (10-100) of detected boxes by the bottom-up attention networks tweaking by some hyperparameters I guess. I updated those numbers to match with the released data in 54f044c.

**Comment by jaesuny** [comment]
I extracted the flickr bottom-up attention features separately from @jnhwkim.
And as he mentioned, I guess the environment or hyperparameter settings were different from @jnhwkim's work.

Sorry that I think I blew the previous google drive link.
I re-uploaded bottom-up attention feature files corresponding 29906.

https://drive.google.com/file/d/1Rdu8UDEi4Z0lnMaqwQYJKBeNTnSY-dsw/view?usp=sharing

**Comment by jnhwkim** [comment]
@jaesuny thanks for the reply and the link! 
I keep the commit 54f044c since it is used for the camera-ready (I believe). Please refer to @jaesuny 's data to double-check if you consider it.

**Comment by ziyanyang** [comment]
Thank you for your help @jnhwkim @jaesuny !
